### PR TITLE
Day type on work session - Creating structure

### PR DESCRIPTION
**Pull request by bitamar**
#191

### CODE REVIEW

File: productivity/modules/custom/productivity_timewatch/productivity_timewatch.features.user_role.inc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * @file
+ * productivity_timewatch.features.user_role.inc
+ */
+
+/**
+ * Implements hook_user_default_roles().
+ */
+function productivity_timewatch_user_default_roles() {
```
Comment (bricel) [inline productivity/modules/custom/productivity_timewatch/productivity_timewatch.features.user_role.inc]: remove this

File: productivity/modules/custom/productivity_migrate/csv/taxonomy_term/session_types.csv [hunk 2]
```diff
@@ -0,0 +1,9 @@
+ID,Parent,Name,Description,Full day
+1,,Regular,,
+2,,Vacation,,
+3,,Sick,,
+4,,Holiday,,1
+5,,Funday,,1
+6,,Miluim,,1
+7,,Convention,,
+8,,Off,,1
```
Comment (bricel) [inline productivity/modules/custom/productivity_migrate/csv/taxonomy_term/session_types.csv]: Need to find a better name @OritGeron, what would you call this.

Comment (bitamar) [inline productivity/modules/custom/productivity_migrate/csv/taxonomy_term/session_types.csv]: @bricel @OritGeron - Maybe: `Off - Unpaid` ?

**Comment by bricel** [comment]
Merged with #252.

**Pull request closed without merging**